test/acceptance/features/steps/kubernetescli.py

```diff
@@ -462,11 +458,18 @@ def assert_generation(context, count):
     return context.latest_application_generation - context.original_application_generation == int(count)
 
 
-@step(u'{type}/{resource_name} is available in the cluster')
-def check_resource_exists(context, type, resource_name):
-    kubernetes = context.kubernetes
-    assert kubernetes.is_resource_in(
-        type, resource_name), f"Could not find the resource: '{type}/{resource_name}' in the cluster"
+@step(u'{resource_type}/{resource_name} is available in cluster "{cluster}"')
```

Comment:
aren't quotes needed here?

Reply:
Nope; so long as the parser can unambiguously figure out which rule you're talking about (and assign captured variables correctly), you can structure your rule however you please.